src/main/java/org/dinosaur/foodbowl/domain/photo/ImageUtils.java

```diff
@@ -0,0 +1,86 @@
+package org.dinosaur.foodbowl.domain.photo;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import javax.imageio.ImageIO;
+import org.imgscalr.Scalr;
+import org.imgscalr.Scalr.Mode;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Component
+public class ImageUtils {
+
+    @Value("${file.dir}")
+    private String fileDir;
+
+    public List<String> storeImageFiles(ImageType imageType, List<MultipartFile> files)
+        throws IOException {
+        List<String> storedImagesPaths = new ArrayList<>();
+
+        for (MultipartFile file : files) {
+            storedImagesPaths.add(storeImageFile(imageType, file));
+        }
+        return storedImagesPaths;
+    }
+
+    public String storeImageFile(ImageType imageType, MultipartFile file) throws IOException {
+        if (file.isEmpty()) {
+            throw new FileNotFoundException("파일이 존재하지 않습니다.");
+        }
+        String originalFilename = file.getOriginalFilename();
+        String storeFileName = createStoreFileName(originalFilename);
```

Comment:
```suggestion
        String storeFilename = createStoreFilename(originalFilename);
```
`file`의 메서드처럼 `createStoreFilename`으로 가는건 어떠신가요?

Reply:
`FileName`인줄 알았는데 `Filename` 이네용!!